include/picongpu/plugins/PhaseSpace/PhaseSpace.hpp

```diff
@@ -102,6 +106,52 @@ namespace picongpu
         void pluginUnload();
     };
 
-}
+namespace particles
+{
+namespace traits
+{
+    template<
+        typename T_Species,
+        typename T_AssignmentFunction
+    >
+    struct SpeciesEligibleForSolver<
+        T_Species,
+        PhaseSpace<
+            T_AssignmentFunction,
+            T_Species
+        >
+    >
+    {
+        using FrameType = typename T_Species::FrameType;
+
+        using RequiredIdentifiers = MakeSeq_t<
+            weighting,
+            position,
```

Comment:
@psychocoderHPC when I chane this to `position` I would need a `pmacc::traits::HasAlias` trait but we only have a `HasIdentifiers` trait :(

Reply:
or can I write `position<>`?

Reply:
Ohh sry my fault you need to write `position<>`

Reply:
yes, that works :)